### ISSUE TITLE
Update Accounts page with no accounts

### ISSUE THREAD

**Issue by jens-concordium** [Concordium/concordium-reference-wallet-android#28]
**Description**
After the implementation of #24, a new issue has come to light. It is likely also related to an old feature, that has been "abandoned" for a while. 

The introduction of the new "failed identities flow", now requires the user to be able to revisit failed identity cards. This means, that if the very first initial account / identity pair created in the wallet fails, the user cannot be sent back to the intro flow, that is shown on first launch of the app. This has been implemented as intended in #24, but since the list of accounts page is now empty, a "Create account" button is now shown.

This button is probably a leftover from before Identities were introduced in the mobile wallet.

What needs to be changed now, is that instead of leading to the "Create account flow" it should lead to the "Create identity and initial account flow". In practice this button should only be shown, when there are no successful identities in wallet, meaning that there are no accounts on the Accounts page at all. 

If the user is waiting for the first initial account / identity pair to be verified on the Accounts page, the "Create account" button that appears in the current version of the app, also only appears when the Accounts page is "refreshed" by browsing away from the Accounts page and back again. The button should instead be shown immediately after pressing "Cancel" on the "failed identity dialog".

A video of the scenario can be found here:

https://user-images.githubusercontent.com/76045015/133993927-d24ef0fa-0af4-4f03-9093-36d2e1f40182.mov

**Steps to Reproduce**
1. Install a version of the Android wallet that contains #24, e.g. 1.0.10 for Testnet.
2. Start creating a new identity and initial account pair via the intro flow. Make sure to submit something that will fail, e.g. use the backside of a "rejsekort" or similar, when prompted to submit identity card or drivers license by Notabene.
3. Wait for the identity to fail while looking at the Accounts page. Press Cancel on the dialog that appears after #24 was implemented.
4. Notice how nothing is shown on the Accounts page.
5. Browse to the Identities page, and then back to the Accounts page.
6. Notice how there is now a button called "Called account"

**Expected Result**
After the "failed identity dialog" shown after the very first initial account / identity pair is shown in the wallet, and the user presses "Cancel", there should be a button on the empty Accounts page. This button should be named "Create initial account" and should lead to the identity / initial account flow.

**Actual Result**
If the user is looking at the Accounts screen when the error dialog appears, the button will not appear before browsing away and back again. The button also says "Create account" and leads to the "regular" create account flow.

**Versions**
- Software Version: 1.0.10 for Testnet
- OS: Android 10
- Mobile device: Huawei P30 Lite
